yt_dlp/extractor/wrestleuniverse.py

```diff
@@ -113,7 +114,7 @@ def _call_api(self, video_id, param='', msg='API', auth=True, data=None, query={
         if data:
             headers['Content-Type'] = 'application/json;charset=utf-8'
             data = json.dumps(data, separators=(',', ':')).encode()
-        if auth and self._TOKEN:
+        if auth:
```

Comment:
This was fine to do; for the best even. Sorry if I was being confusing w/my earlier comment
```suggestion
        if auth and self._TOKEN:
```